fs/copy.py

```diff
@@ -101,13 +155,14 @@ def copy_dir(src_fs, src_path, dst_fs, dst_path, walker=None):
         in ``src_fs``. Set this if you only want to consider a sub-set
         of the resources in ``src_fs``.
     :type walker: :class:`~fs.walk.Walker`
+    :returns: a list with the copied files dst_path.
```

Comment:
Yes. Works for me. Only suggestion I would have is to rename `callback` to a more descriptive `on_copy`.